docs/DeveloperGuide.md

```diff
@@ -24,43 +29,44 @@ If you have problems, see <a href="#debug">Debugging</a>
    ```bash
    make docker_build DLDT_PACKAGE_URL=<URL>
    ```
-   Note: URL to OpenVINO Toolkit package can be received after registration on [OpenVINO™ Toolkit website](https://software.intel.com/en-us/openvino-toolkit/choose-download)
-2. Mount the source code in the Docker container:
+   > **Note**: URL to OpenVINO Toolkit package can be received after registration on [OpenVINO&trade; Toolkit website](https://software.intel.com/en-us/openvino-toolkit/choose-download)
+2. Mount the source code in the Docker container :
 	```bash
 	docker run -it -v ${PWD}:/ovms --entrypoint bash -p 9178:9178 openvino/model_server-build:latest 
 	```
 
-3. In the docker container context compile the source code via:
+3. In the docker container context compile the source code via :
 	```bash
 	bazel build //src:ovms
 	```
 
-4. From the container, run a single unit test:
+4. From the container, run a single unit test :
 	```bash
 	bazel test --test_summary=detailed --test_output=all --test_filter='ModelVersionStatus.*' //src:ovms_test
 	```
 
-	where:
+| Argument      | Description |
+| :---        |    :----   |
+| `test`       | builds and runs the specified test target       |
+| `--test_summary=detailed`   |   the output includes failure information       |
+| `--test_output=all` | log all tests |
+| `--test_filter='ModelVersionStatus.*'` | limits the tests run to the indicated test  | 
+| `//src:ovms_test` | the test source |
+> **NOTE**: For more information, see the [bezel command-line reference](https://docs.bazel.build/versions/master/command-line-reference.html)
```

Comment:
@mandar-opc please correct (bazel not bezel). Thank you.

Reply:
This is fixed.